Build Process Error for AdaptiveCpp

# Summary
While building oneMath with AdaptivCpp , I am getting the following error.

oneMath/src/blas/backends/cublas/cublas_helper.hpp:263:5: error: static assertion failed
    static_assert(false);
    ^             ~~~~~
1 error generated when compiling for sm_70.


# Environment
Linux, x86_64
cuda:sm_70

# Steps to reproduce
git clone: https://github.com/uxlfoundation/oneMath.git
cmake .. -DONEMATH_SYCL_IMPLEMENTATION=hipsycl  
        -DENABLE_MKLGPU_BACKEND=False 
        -DENABLE_MKLCPU_BACKEND=False
        -DENABLE_CURAND_BACKEND=True       
        -DENABLE_CUBLAS_BACKEND=True    
        -DHIPSYCL_TARGETS=omp
        -DHIPSYCL_TARGETS=cuda:sm_70
        -DBUILD_FUNCTIONAL_TESTS=False         
        -DBUILD_EXAMPLES=False   

cmake --build .

Thank You.

--- a/src/blas/backends/cublas/cublas_helper.hpp
+++ b/src/blas/backends/cublas/cublas_helper.hpp
@@ -260,7 +260,7 @@ inline cublasSideMode_t get_cublas_side_mode(oneapi::math::side lr) {
 
 template <typename T>
 inline cudaDataType_t get_cublas_datatype() {
-    static_assert(false);
+    static_assert(sizeof(T) && false, "Type T has no corresponding cuBLAS type");
 }
 
 template <>

--- a/src/blas/backends/rocblas/rocblas_helper.hpp
+++ b/src/blas/backends/rocblas/rocblas_helper.hpp
@@ -218,7 +218,7 @@ inline rocblas_side get_rocblas_side_mode(oneapi::math::side lr) {
 
 template <typename T>
 inline rocblas_datatype get_rocblas_datatype() {
-    static_assert(false);
+    static_assert(sizeof(T) && false, "Type T has no corresponding rocBLAS type");
 }
 
 template <>

